encrypt-core/encrypt-core-common/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptRule.java

```diff
@@ -210,7 +210,7 @@ public boolean isCipherColumn(final String tableName, final String columnName) {
      * @return logic and cipher columns
      */
     public Map<String, String> getLogicAndCipherColumns(final String logicTable) {
-        return tables.containsKey(logicTable) ? tables.get(logicTable).getLogicAndCipherColumns() : Collections.emptyMap();
+        return tables.containsKey(logicTable.toLowerCase()) ? tables.get(logicTable.toLowerCase()).getLogicAndCipherColumns() : Collections.emptyMap();
```

Comment:
return Optional.ofNullable(tables.get(logicTable)).map(EncryptTable::getLogicAndPlainColumns).orElse(Collections.emptyMap());

should be better.

Reply:
OK, I fix it.